project-code/core/common/src/main/scala/play/core/server/servlet/RequestHandler.scala

```diff
@@ -102,6 +102,10 @@ trait HttpServletRequestHandler extends RequestHandler {
       case (name, value) if name.equalsIgnoreCase(play.api.http.HeaderNames.SET_COOKIE) =>
         getServletCookies(value).foreach(httpResponse.addCookie)
 
+      case (name, value) if name.equalsIgnoreCase(HeaderNames.TRANSFER_ENCODING) && value == HttpProtocol.CHUNKED =>
+        // ignore this header
+        // the JEE container sets this header itself. Avoid duplication of header (issues/289)
```

Comment:
actually, it is working because the header is duplicated. But most of HTTP clients deal with that.
It is only a trouble for some picky clients... ;)

As far as I know, the containers set this header automatically.
But I have not searched the JEE specs to check if the container must set this header.

It is a minor issue in my opinion.